src/filter/median.rs

```diff
@@ -242,12 +256,15 @@ impl HistSet {
             data.push([0u32; 256]);
         }
 
-        HistSet { data, expected_count }
+        HistSet {
```

Comment:
What's the rule that's being applied here? This line is already very short. Does rustfmt now enforce one row per field for all struct initialisations?

Reply:
Uh, indeed, it's aggressive a bit. I think the reason is here https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#struct. I added `#[rustfmt::skip]`

Reply:
Thanks for the link. I think we should just go with the default here, even if it's a bit odd.